contracts/staking/StakingPool.sol

```diff
@@ -228,14 +230,24 @@ contract StakingPool is
         _withdrawStake();
     }
 
+    function sweepERC20Tokens(address tokens, uint256 amount)
+        external
+        whenNotPaused
+        onlySuperUserRole
```

Comment:
this permissioning will change to onlyOwner

Reply:
👍